src/app/modules/forms/editor/editor.component.ts

```diff
@@ -0,0 +1,64 @@
+import {Component} from '@angular/core';
+import {FormBuilder, FormGroup, ReactiveFormsModule, Validators} from "@angular/forms";
+import {MatButton, MatIconButton} from "@angular/material/button";
+import {MatCard, MatCardContent, MatCardTitle} from "@angular/material/card";
+import {MatOption} from "@angular/material/core";
+import {MatDatepicker, MatDatepickerInput, MatDatepickerToggle} from "@angular/material/datepicker";
+import {MatDialogRef} from "@angular/material/dialog";
+import {MatFormField, MatLabel, MatSuffix} from "@angular/material/form-field";
+import {MatIcon} from "@angular/material/icon";
+import {MatInput} from "@angular/material/input";
+import {MatSelect} from "@angular/material/select";
+
+@Component({
+  selector: 'app-editor',
+  standalone: true,
+  imports: [
+    MatButton,
+    MatCard,
+    MatCardContent,
+    MatCardTitle,
+    MatDatepicker,
+    MatDatepickerInput,
+    MatDatepickerToggle,
+    MatFormField,
+    MatInput,
+    MatLabel,
+    MatOption,
+    MatSelect,
+    MatSuffix,
+    ReactiveFormsModule,
+    MatIcon,
+    MatIconButton
+  ],
+  templateUrl: './editor.component.html',
+  styleUrl: './editor.component.css'
+})
+export class EditorComponent {
```

Comment:
do you really need EditorComponent if you have a creation one? they are similar)

Reply:
Interesting, I can delete it, but then I have to change the logic that I planned to add, instead of loading different forms for the task, I can dynamically change some of the functionality, but then the component does not fully comply with the principle of single responsibility

Reply:
but the logic in homogenic - the only difference is the fact you have the data or the form is empty :)